validation/standard/standard-versions.toml

```diff
@@ -1,14 +1,38 @@
-["op-contracts/v1.4.0"] # Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
-address_manager = ""
-l1_cross_domain_messenger = "2.3.0"
-l1_erc721_bridge = "2.1.0"
-l1_standard_bridge = "2.1.0"
-optimism_mintable_erc20_factory = "1.9.0"
-optimism_portal = "3.10.0"
-system_config = "2.2.0"
-protocol_versions = "1.0.0"
-anchor_state_registry = "1.0.0"
-delayed_weth = "1.0.0"
-dispute_game_factory = "1.0.0"
-fault_dispute_game = "1.2.0"
-permissioned_dispute_game = "1.2.0"
+op_contracts_release = "op-contracts/v1.4.0"
+
+[contract_tag]
+
+# Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
+[contract_tag."op-contracts/v1.4.0"]
+address_manager = { version = "" }
+l1_cross_domain_messenger = { version = "2.3.0" }
+l1_erc721_bridge = { version = "2.1.0" }
+l1_standard_bridge = { version = "2.1.0" }
+optimism_mintable_erc20_factory = { version = "1.9.0" }
+optimism_portal = { version = "3.10.0" }
+system_config = { version = "2.2.0" }
+protocol_versions = { version = "1.0.0" }
```

Comment:
`protocol_versions` is a superchain level contract, like SuperchainConfig, so we don't need that here